src/main/java/org/ossgang/commons/observables/ExceptionHandlers.java

```diff
@@ -0,0 +1,31 @@
+package org.ossgang.commons.observables;
+
+import java.util.Objects;
+import java.util.function.Consumer;
+
+final class ExceptionHandlers {
+    private static volatile Consumer<Exception> uncaughtExceptionHandler = ExceptionHandlers::printExceptionToStderr;
```

Comment:
I somehow overlooked this detail in the previous version also.. do you think we should have a `Consumer<Throwable>`? So we are in sync with all the other method signatures :smile:

Reply:
Indeed Good question (also for the other signatures, shall we use Exception or Throwable?)...

Let's change in a new PR.